### DISCUSSION TITLE
Probabilities of each record for being relevant?

### DISCUSSION THREAD

**Discussion by SagevdBrand** [asreview/asreview#546] [Q&A] [answered]
Is there a way to see the the probabilities of each record as being relevant?

**Comment by J535D165** [comment, marked as answer]
Short answer: This is not possible ASReview LAB users. 

ASReview doesn't output probabilities because:

1. Not all models are based on probabilities (SVM for example).
2. Probabilities in the Active Learning cycle can be really hard to interpret. ASReview makes use of advanced (re)sampling techniques and query strategies. As a result of this, probabilities are not unbiased. This makes ASReview very efficient in finding relevant records, but the probabilities are hard to interpret for a user. 

When exporting the results of your review in ASReview (https://asreview.readthedocs.io/en/latest/features/post_screening.html#download-results), the unlabeled records are ranked on the order in which they would have been presented to the user. One can consider this as a ranking. Important: this is true for the default settings, but not for all classifiers/query strategies.
